src/gui/folderwatcher.cpp

```diff
@@ -57,7 +58,7 @@ bool FolderWatcher::pathIsIgnored( const QString& path )
 
 #ifndef OWNCLOUD_TEST
     if (_folder->isFileExcludedAbsolute(path)) {
-        qDebug() << "* Ignoring file" << path;
+        qCDebug(lcFolderWatcher) << "* Ignoring file" << path;
```

Comment:
I'm undecided on this one, but think it should be shown all the time.
@ogoffart @ckamm opinions?